rllib/tuned_examples/dqn/multi_agent_cartpole_dqn.py

```diff
@@ -63,4 +65,6 @@
 if __name__ == "__main__":
     from ray.rllib.utils.test_utils import run_rllib_example_script_experiment
 
+    args.local_mode = True
```

Comment:
nit: no local mode

On enable_new_api_stack:
Either "bake" it into the config below (config.api_stack(...)) 
OR force - through an assert - the user to provide the `--enable-new-api-stack`.